README.md

```diff
@@ -4,7 +4,8 @@
 - Extends the functionality of the [EU Cookie compliance contrib module](https://www.drupal.org/project/eu_cookie_compliance).
 - Provides a block containing the same content as the EU Cookie popup.  When this block is added to a page, we get a dedicated Cookie settings page.  At the time of writing in early 2023, many sites including the [BBC](https://www.bbc.co.uk/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/) use such a dedicated Cookie settings page.  This is the primary feature of this module.
 - Only relevant when you want users to have greater control over site cookies by enabling them to accept or reject certain types of cookies e.g. accept Functional cookies, reject Analytics cookies and so on.
-- <a name="secondary-feature"></a>The EU Cookie compliance module actually doesn't stop any cookie being set prior to consent.  What it does is, it removes unwanted cookies immediately after they are set.  The custom module improves up on this for Google analytics and Hotjar in a way that these modules would not even set any cookie in the browser prior to consent.  This is a secondary feature of this module.
```

Comment:
@markconroy I've reworded this description of the Analytics loader to be clearer that this module offers javascript blocking as well as just cookie cleardown, and include how to configure that.

Reply:
Far far better than anything I have written.  Thanks a ton.